docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     environment:
       POSTGRES_USER: user
       POSTGRES_PASSWORD: password
-      POSTGRES_DB: fittracker
+      POSTGRES_DB: 'fittracker'
```

Comment:
Changed it while trying to make `fittracker` schema appear in `fittracker` database instead of `postgres` maintenance one. Turned out to not be the problem.